docs/docs/api/06-Configuration/askui-client.md

```diff
@@ -0,0 +1,82 @@
+---
+custom_edit_url: null
+---
+# askui Client
```

Comment:
the client is now called AskuiClient is now called UiControlCLient.
should this be renamed to askui UI Control Client?